### ISSUE TITLE
Fix Request: LetFieldsMeetSerializable has false positives

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#231]
```
@Data
public abstract class SomeDTO implements Serializable {
    private static final long serialVersionUID = 1;
    @Min(1)
    private Long id;
    @NotNull
    private AType aType;
    private List<@Valid ValMappingDTO> valMappings;
//Fields in a Serializable class should be serializable or transient.
    private List<@Valid TmplMappingDTO> tmplMappings;
//Fields in a Serializable class should be serializable or transient.
}
```
While ValMappingDTO and TmplMappingDTO implement Serializable.
